src/panels/lovelace/cards/hui-thermostat-card.ts

```diff
@@ -129,7 +129,7 @@ export class HuiThermostatCard extends LitElement implements LovelaceCard {
             ></round-slider>
           `;
 
-    const currentTemperature = stateObj.attributes.current_temperature
+    const currentTemperature = !isNaN(stateObj.attributes.current_temperature)
```

Comment:
`currentTemperature` is a boolean now?

Reply:
No, it is a svg or nothing, this is the check

Reply:
https://github.com/home-assistant/home-assistant-polymer/blob/39f72da115a9ff0a55796971d878f730e3cd792a/src/panels/lovelace/cards/hui-thermostat-card.ts#L230